### PR TITLE
[doc] Refactor text about taint on the fifths step in the bare metal section

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

The context of either removing taint or adding nodes to the cluster has been strengthened.

## Why do we need it, and what problem does it solve?

Now this moment does not look obvious, and users often miss the need for these actions.

## Why do we need it in the patch release (if we do)?

To reduce possible confusion in for a reader.

## What is the expected result?

The Getting Started is consistent and clean.

## Checklist

- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: getting started
type: chore
summary: Docs became more consistent.
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md [hunk 1]
```diff
@@ -4,18 +4,56 @@
...
+Запуск всех компонентов Deckhouse после завершения установки может занять какое-то время.
 </blockquote>
 
-Запуск всех компонентов Deckhouse после завершения установки может занять какое-то время. 
+<ul>
+  <li>
+    Подготовьте <strong>чистую</strong> виртуальную машину, которая будет узлом кластера.
+  </li>
+  <li>
+    Создайте <a href="http://ru.localhost/documentation/v1/modules/040-node-manager/">NodeGroup</a> <code>worker</code>. Для этого выполните на master-узле:
```
Comment (z9r5) [inline docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md:26]:
```suggestion
    Создайте <a href="/documentation/v1/modules/040-node-manager/cr.html#nodegroup">NodeGroup</a> <code>worker</code>. Для этого выполните на master-узле:
```

File: docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md [hunk 2]
```diff
@@ -4,18 +4,56 @@
...
+    {% snippetcut %}
+  ```bash
+kubectl create -f - << EOF
```
Comment (z9r5) [inline docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md:29]:
```suggestion
sudo /opt/deckhouse/bin/kubectl create -f - << EOF
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Pull request merged**